aries_cloudagent/messaging/jsonld/routes.py

```diff
@@ -95,40 +128,49 @@ async def verify(request: web.BaseRequest):
     response = {"valid": False}
     try:
         context: AdminRequestContext = request["context"]
+        profile = context.profile
         body = await request.json()
         verkey = body.get("verkey")
         doc = body.get("doc")
-
         async with context.session() as session:
-            wallet = session.inject(BaseWallet, required=False)
-            if not wallet:
-                raise web.HTTPForbidden(reason="No wallet available")
-            valid = await verify_credential(doc, verkey, wallet)
-
+            if verkey is None:
+                resolver = session.inject(DIDResolver)
+                ver_meth_expanded = await resolver.dereference(
+                    profile, doc["proof"]["verificationMethod"]
+                )
+                if ver_meth_expanded is None:
+                    raise MissingVerificationMethodError(
+                        f"Verification method "
+                        f"{doc['proof']['verificationMethod']} not found."
+                    )
+                verkey = ver_meth_expanded.material
+            valid = await verify_credential(session, doc, verkey)
         response["valid"] = valid
-    except (BadJWSHeaderError, DroppedAttributeError) as e:
+    except (
+        BaseJSONLDMessagingError,
+        ResolverError,
+    ) as e:
         response["error"] = str(e)
-
+    except (WalletError, InjectionError):
+        raise web.HTTPForbidden(reason="No wallet available")
     return web.json_response(response)
 
 
 async def register(app: web.Application):
     """Register routes."""
 
-    app.add_routes([web.post("/jsonld/sign", sign)])
-    app.add_routes([web.post("/jsonld/verify", verify)])
+    app.add_routes([web.post("/jsonld/sign", sign), web.post("/jsonld/verify", verify)])
 
 
 def post_process_routes(app: web.Application):
     """Amend swagger API."""
-
     # Add top-level tags description
     if "tags" not in app._state["swagger_dict"]:
         app._state["swagger_dict"]["tags"] = []
     app._state["swagger_dict"]["tags"].append(
         {
             "name": "jsonld",
-            "description": "JSON-LD message operations",
+            "description": "sign and verify json-ld data.",
```

Comment:
Conventionally, throughout aca-py the description starts with a capital but has no trailing period.

Reply:
I see. I will go and update the descriptions to follow conventions.